model/feature-flag/logs.yaml

```diff
@@ -1,11 +1,26 @@
 groups:
-  - id: log-feature_flag
+  - id: feature_flag.evaluation
     type: event
-    stability: experimental
-    name: feature_flag
+    name: feature_flag.evaluation
     brief: >
-      This event describes feature flag evaluation representation on a Log Record.
+      This semantic convention defines the attributes used to represent a
```

Comment:
can we describe an event here (or in the note) - e.g. who/how/when someone would report it?

Reply:
Would you want that here or in logs.yaml?

Reply:
nvm i see this comment is on logs.yaml. I was turned around.